test/integration/features/basic.feature

```diff
@@ -44,6 +44,15 @@ Feature: Basic
     minishift
     """
 
+  Scenario: Persistent volumes should be listed when minishift start
+    When executing "oc get pv --as system:admin -o=name"
+    Then stderr should be empty
+     And exitcode should equal 0
+     And stdout should contain
+     """
+     persistentvolume/pv0001
+     """
```

Comment:
@LalatenduMohanty That's a very basic test, deployment a app using a PV might not work for this moment because app deployment still issue for use due to that waiting time.